app/views/ratings/edit.html.erb

```diff
@@ -1,5 +1,6 @@
+<%= link_to('<- Back', :back) %>
```

Comment:
This single back button takes the user back to where they were without making assumptions about where that was.